project/MimaExcludes.scala

```diff
@@ -34,6 +34,20 @@ import com.typesafe.tools.mima.core.ProblemFilters._
 object MimaExcludes {
     def excludes(version: String) =
       version match {
+        case v if v.startsWith("1.5") =>
+          Seq(
+            MimaBuild.excludeSparkPackage("deploy"),
+            // These are needed if checking against the sbt build, since they are part of
+            // the maven-generated artifacts in 1.3.
+            excludePackage("org.spark-project.jetty"),
```

Comment:
It was actually failing before I added it. Probably worth digging into, but given all the races with this patch I am inclined to maybe merge this in now and then worry about it later.